pkg/agent/client/util.go

```diff
@@ -32,22 +31,12 @@ func registrationEntryFromProto(e *types.Entry) (*common.RegistrationEntry, erro
 	if e.Id == "" {
 		return nil, fmt.Errorf("missing entry ID")
 	}
-	parentID, err := spiffeIDFromProto(e.ParentId)
```

Comment:
I'm wondering if we should rename the `registrationEntryFromProto` function to reflect the fact that only the "essential" fields are returned. Otherwise could be confusing if in the future this function is called in other places expecting to map all the fields.

Reply:
Absolutely, although this is not a public function the name can be misleading. Does `slicedEntryFromProto` sound like a reasonable name?

Reply:
Yes, I think that's a better name.